### DISCUSSION TITLE
Text is off center on the new architecture

### DISCUSSION THREAD

**Discussion by okwasniewski** [reactwg/react-native-new-architecture#81] [Q&A] [answered]
Hey, 

I'm trying to migrate `react-native-paper` to the new architecture, removing deprecated API's. I came across an issue where Text is off center in few of our components. 

![CleanShot 2022-10-05 at 10 14 42@2x](https://user-images.githubusercontent.com/52801365/194014092-17b48c9b-d9f1-476d-b4af-c433defb16d5.png)

Next I've checked the native elements with Flipper: 

### Old Architecture
<img width="1102" alt="CleanShot 2022-10-04 at 22 37 23@2x" src="https://user-images.githubusercontent.com/52801365/194014641-c5c71e81-fa69-49d5-8a0f-af3331f6b212.png">

### New architecture
![CleanShot 2022-10-04 at 22 43 13@2x](https://user-images.githubusercontent.com/52801365/194014775-352b5593-7bec-49d9-b337-db58da645874.png)

Tree structure is different which is expected due to view flattening, however this new `RCTParagraphComponentView` behaves differently aligning the text to the bottom not to center. It's also worth mentioning that the dimensions of native elements are the same. Also JS code didn't change between old and new architecture.

Here is the code for this component: https://github.com/callstack/react-native-paper/blob/111a034e36431575bb40f8155367e39ccd9e63ba/src/components/Chip/Chip.tsx#L326

**Comment by sammy-SC** [comment, marked as answer]
Hello @okwasniewski,

thank you for the report. Would it be possible to extract this into minimal repro case?

**Reply by sammy-SC** [reply]
I think this might be issue with lineHeight. If that's the case, I'm working on a fix.

**Reply by sammy-SC** [reply]
Could you try if https://github.com/facebook/react-native/commit/70cc27c901aeb447910e30ac3ceac85990d3c16d fixes it?

If not, please let me know.

**Reply by okwasniewski** [reply]
Thanks for fast response. I've tried latest nightly release and it looks like it fixes our issue 🎉

**Reply by sammy-SC** [reply]
Thank you @okwasniewski
